src/System.DirectoryServices/tests/System/DirectoryServices/ActiveDirectorySecurityTests.cs

```diff
@@ -6,6 +6,7 @@
 
 namespace System.DirectoryServices.Tests
 {
+    [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot, "DirectoryObjectSecurity is not supported.")]
```

Comment:
This is redundant
https://github.com/dotnet/buildtools/blob/master/src/xunit.netcore.extensions/TargetFrameworkMonikers.cs#L25
But, harmless